Lib/test/test_capi/test_object.py

```diff
@@ -223,5 +224,16 @@ def __del__(self):
             obj = MyObj()
             _testinternalcapi.incref_decref_delayed(obj)
 
+    def test_is_unique_temporary(self):
+        self.assertTrue(_testcapi.pyobject_is_unique_temporary(object()))
+        obj = object()
+        self.assertFalse(_testcapi.pyobject_is_unique_temporary(obj))
+
+        def func(x):
+            self.assertEqual(sys.getrefcount(x), 1)
```

Comment:
```suggestion
            # This relies on the LOAD_FAST_BORROW optimization (gh-130704)
            self.assertEqual(sys.getrefcount(x), 1)
```

Probably a good idea to add a comment to aid future debugging when that optimisation breaks. "If", I mean. I mean "if" that optimisation breaks.